user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+functionalities. For example, you can limit the users' abiltiy to assign tags, share grids, or create new users.
+
+For each role and capability pair, there are only two options:
+
+- **None**: users with this role won’t be able to use the relevant functionality.
+- ***System***: users with this role will be able to use the relevant functionality for all the records created within 
+  the OroCRM instance they’ve logged into..
+
+  
+.. image:: ./img/roles/01.png   
+  
+This guide describes, in alphabetical order, the set of capabilities that are available and their related 
+functionalities, and provides examples of several default capability settings for some common roles.
+
+
+
+.. _admin-capabilities-acc:
+
+Abandoned Cart Campaign Manipulations
+-------------------------------------
+
+If this capability is set to "System," the user can use the *Abandoned Cart Campaign* functionality. With the functionality
+you can generate automatic e-mailings to customers in your Magento store who have not completed their purchase in order to 
+convince them to complete the checkout and place an order. You can find more details about this functionality in the
+:ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.
+
+This functionality is useful for marketing representatives and any other employees who are working with email campaigns 
+directed at customers of Magento-based store.
+
+.. important::
+
+    To use the functionality, you also need to :ref:`set up the Abandoned Cart extension <user-guide-acc-ext>`.
+
+    
+.. _admin-capabilities-data-audit:    
+    
+Data Audit
+----------
+
+If this capability is set to *"System"*, the user can see the full history of changes made to any record of an
+:ref:`auditable entity <user-guide-entity-management-create-other>`, as well as an out-of-the-box report of all such 
+actions. 
+
+The report is available from the "*System*" menu and described in more details in the 
+:ref:`Data Audit <user-guide-data-audit>` guide.
+
+The link to a specific record's history of changes is available in the top right corner of the record’s
+:ref:`View page <user-guide-ui-components-view-pages>`.
+
+|
+
+.. image:: ./img/data_management/view/view_history.png
+
+|
+
+This Data Audit is usually used by system administrators, but may also be used by managers in order to see how details 
+of different records were changed, as well as by whom.
+
+
+.. _admin-capabilities-dotmailer-stats:    
+
+Dotmailer Statistic
+-------------------
+
+This capability is meaningful if the system has been integrated with dormatiler. 
+
+Such integration lets users utilize dotmailer as an email campaign engine, as described in the guide on
+:ref:`Integration with Dotmailer <user-guide-dm-integration>`. So, the user can synchronize lists of contacts in OroCRM 
+and dotmailer, and use them to create email campaigns in dotmailer and import them to OroCRM. 
```

Comment:
Edit: Such integration lets users utilize dotmailer as an email campaign engine, as described in the guide on :ref:`Integration with Dotmailer <user-guide-dm-integration>`. In short, the user can synchronize lists of contacts in OroCRM and dotmailer, and use them to create email campaigns in dotmailer and import them to OroCRM.